README.md

```diff
@@ -189,37 +174,6 @@ cd PVM && git pull
 sudo reboot
 ```
 
-## Examples
-
-Navigate to the `examples` folder and open the `examples.maxproj` file.
-
-A series of quick examples appears listed on the Max project window.
-
-Patch `#00.maxpat` serves as an index of the examples provided.
-
-Given the objective of making this project available to creative artists, the control interface was created as a Max project. The patch `pvm.maxpat` that opens automatically contains several objects and different abstractions that work together to generate/format and send the control messages to the PVM devices. The `pvm_init` patcher should automatically open and parse the `max-init.txt` file, and configure the multiple `pvm_send` patchers. If some changes were made or reloading is required, simply click the `reset` message box to retrigger this. 
-
-At this point, the `pvm_control` bpatcher, on the top left portion, should be able to control all devices at the same time. The available controls should be mostly self explanatory: 
-
-i) the toggle at top left works like a start/stop button; 
-
-ii) the `next_frame` message box triggers the next frame of the video when it is paused; 
-
-iii) the `fullscreen` message box toggles the fullscreen display;
-
-iv) the two number boxes below change (and trigger) the `set_position` and `set_rate` commands. 
-
-Should you want to control just one of the devices or a sub-selection, feel free to delete patch chords accordingly. To operate the warmup system at the bottom right, consisting of the `pvm_warmup` bpatcher, first dial some value in all the four number boxes and then press the toggle to activate it. 
-
-The (proof of concept) resync system, on the top right portion of the patch, includes a toggle that enables a metro object to force the devices to go to position 0. (i.e. start of the clip) every `n` seconds, where `n` is determined by the number box just above.
```

Comment:
again: https://github.com/omarcostahamido/PVM/issues/57#issuecomment-1328413740

Reply:
Add example section back and change wiki page
https://github.com/omarcostahamido/PVM/wiki/The-workflow-in-MAX